### ISSUE TITLE
Update & expand Playwright/axe a11y test coverage

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/bedrock#16690]
## One-line summary
Resets some exclusion jokes, and expands the url list for a11y coverage.

## Significant changes and points to review
There are axe test failures on some of the newly added pages. I'll add the reports for more info. This can either land to deliberately start failing the tests until fixed, or can be added with the exclusions enumerated for now, or can wait to merge until the a11y issues reported are fixed.

## Issue / Bugzilla link
#427

## Testing
`npm run a11y-tests`

[test-results.zip](https://github.com/user-attachments/files/22050019/test-results.zip)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/WT-132)
┆Priority: P3
┆Sprint: Backlog
